en/docs/learn/customizing-automated-emails.md

```diff
@@ -109,6 +129,26 @@ template.
         Before sending the email, the WSO2 IS will retrieve the user details
         from the user store configured for this tenant and replace the
         placeholders with the corresponding values of the user.
+        
+        !!! note
+            When you are using an identity claim as a placeholder in the **Account Conifrmation** template
+            for User Self Registration, you need to use the notation ```{{claimName}}``` instead of using 
+            ```{{user.claim.identity.claimName}}```. And additionally you need to send the respsective value 
+            for the placeholder in the API request payload, under the ```properties``` section. 
+            See the example request payload below.
```

Comment:
why have we repeated the same note again here?

Reply:
Since the "Placeholders and Custom Claim Dialects" tip is in two places.
This note is relevant to the above tip.